examples_utils/benchmarks/run_benchmarks.py

```diff
@@ -452,7 +469,7 @@ def run_benchmarks(args: argparse.ArgumentParser):
         json.dump(results, json_file, sort_keys=True, indent=2)
 
     # Parse summary into CSV and save in logs directory
-    csv_metrics = ["throughput", "latency", "total_compiling_time"]
+    csv_metrics = ["throughput", "latency", "total_compiling_time", "loss", "result", "cmd", "env", "git_commit_hash"]
```

Comment:
I think for notebooks we are quite interested in total runtime as well... 😬

Reply:
I'll look into this

Reply:
Nice! Thanks